bluemira/equilibria/optimisation/constraint_funcs.py

```diff
@@ -389,11 +413,16 @@ def cs_fz(self, f_matx):
     def pf_z_constraint(self, f_matx, max_value):
         """Constraint Function Absolute vertical force constraint on PF coils."""
         scaled_max_value = max_value / self.scale
-        self.constraint[: self.n_PF] = f_matx[: self.n_PF, 1] ** 2 - scaled_max_value**2
+        self.constraint[: self.n_PF] = (
+            np.sqrt(f_matx[: self.n_PF, 1] ** 2) - scaled_max_value
```

Comment:
How does this change make a difference. Would the previous case have the same profile as far as the optimiser is concerned? Also instead of squaring and rooting wouldnt the absolute value have the same effect with less calculation.

```suggestion
            np.abs(f_matx[: self.n_PF, 1]) - scaled_max_value
```

I dont know what the right answer is here but as you can tell I'm not quite comfortable with this